botocore/data/devicefarm/2015-06-23/service-2.json

```diff
@@ -4260,7 +4260,7 @@
         },
         "endpoint":{
           "shape":"String",
-          "documentation":"<p>The endpoint for the remote access sesssion.</p>"
+          "documentation":"<p>The endpoint for the remote access session.</p>"
```

Comment:
Would you mind removing this change? Anything under the `/data` directory is generally managed by the service team and will need to be updated upstream to avoid this change being overwritten in the next release.

Reply:
@nateprewitt Sure, I have removed changes from `/data` directory. Thank you for checking out the PR :)